src/include/brain/brain.h

```diff
@@ -28,7 +31,26 @@ namespace brain {
  * the brain, such as RPC and Catalog.
  */
 class BrainEnvironment {
-  // TODO(tianyu): fill in as needed
+ public:
+  BrainEnvironment() :
+      rpc_client_{settings::SettingsManager::GetString(settings::SettingId::peloton_rpc_address)},
+      sql_connection_{settings::SettingsManager::GetString(settings::SettingId::peloton_address)} {}
+
+  inline PelotonService::Client GetPelotonService() {
+    return rpc_client_.getMain<PelotonService>();
+  }
```

Comment:
Is this how we're going to communicate with Peloton server? It does not seem used anywhere.